docs/user-guide/compute-resources.md

```diff
@@ -40,6 +40,32 @@ about pod resource requests/limits.  A *pod resource request/limit* for a partic
 type is the sum of the resource requests/limits of that type for each container in the pod, with
 unset values treated as zero (or equal to default values in some cluster configurations).
 
+### Meaning of CPU
+Limits and requests for `cpu` are measured in cpus.  
+One cpu, in Kubernetes, is equivalent to:
+
+- 1 AWS vCPU
+- 1 GCP Core
+- 1 Azure vCore
+- 1 *Hyperthread* on a bare-metal Intel processor with Hyperthreading
+
+Fractional requests are allowed.  A container with `spec.container[].resources.requests.cpu` of `0.5` will
```

Comment:
Would it be helpful to explicitly state that Kubernetes does not normalize across CPU types with different performance characteristics?